nexus/db-model/src/schema.rs

```diff
@@ -419,6 +419,8 @@ table! {
         active_propolis_id -> Nullable<Uuid>,
         target_propolis_id -> Nullable<Uuid>,
         migration_id -> Nullable<Uuid>,
+        updater_id -> Nullable<Uuid>,
```

Comment:
Whenever `schema.rs` is updated, that probably means we need to update https://github.com/oxidecomputer/omicron/blob/c1374bfed1ad30d35b97261be8259ada944163e3/schema/crdb/dbinit.sql#L970-L1006 too